plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-data.spec.js

```diff
@@ -585,6 +585,103 @@ test.describe( 'Analytics-related tests', () => {
 		).toBeVisible();
 	} );
 
+	test( 'use filter by single product on products report', async ( {
+		page,
+	} ) => {
+		await page.goto(
+			'/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Fproducts'
+		);
+
+		// FTUX tour on first run through
+		try {
+			await page.getByLabel( 'Close Tour' ).click( { timeout: 5000 } );
+		} catch ( e ) {
+			console.log( 'Tour was not visible, skipping.' );
+		}
```

Comment:
I know we are following a different pattern in this file, but could we use [block editor fixtures](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/fixtures/block-editor-fixtures.js#L13-L16) to close the tour?

It would enlarge the scope of this PR a little bit, but I think it's a good improvement to the way we deal with the tours in this file.

We should add something like the following to [this file](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/fixtures/block-editor-fixtures.js):
```
// Disable the analytics tour
await wcAdminApi.post( 'options', {
	woocommerce_revenue_report_date_tour_shown: 'yes',
} );
```

And we also should remove all the `try/catch` that close the tour and replace [this line](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-data.spec.js#L1) with:
```
const { expect } = require( '@playwright/test' );
const { test } = require( '../../fixtures/block-editor-fixtures' );
```

Reply:
Oh, I just saw that the PR has been merged. Mabye we can tackle my suggestion in a follow-up PR.